data/IL/incentives.json

```diff
@@ -18,8 +18,8 @@
       "renter"
     ],
     "short_description": {
-      "en": "$40 rebate for new ENERGY STAR® rated electric clothes dryers with capacity of at least 4.4 cubic feet.",
-      "es": "Un reembolso de $40 dólares para secadoras de ropa eléctricas nuevas con certificación ENERGY STAR® y una capacidad de al menos 4.4 pies cúbicos."
+      "en": "$40 rebate for new ENERGY STAR® rated electric clothes dryers with capacity of at least 4.4 cu. ft.",
+      "es": "Un reembolso de $40 dólares para secadoras de ropa eléctricas nuevas con certificación ENERGY STAR® y una capacidad de al menos 4.4 ft³."
```

Comment:
Hmmm I'm not sure about using ft^3. We can go ahead with it for now, but I'm putting together a little list of stylistic questions I'd like to get straight; will share sometime next week.